wiki/Play_style/Alternating/de.md

```diff
@@ -0,0 +1,28 @@
+---
+tags:
+  - keyboard
+  - tap
+  - alternate
+  - alt
+  - play style
+  - Tastatur
+  - Spielstil
+---
+
+# Alternating
+
+*Nicht zu verwechseln mit [Double-Tapping](/wiki/Play_style/Double_tapping).*
+
+**Alternating** (auf Deutsch *abwechselnd*) ist ein Tippstil für [osu!](/wiki/Game_mode/osu!), bei dem Spieler durch das Alternieren zwischen zwei eingestellten Tasten auf ihrer Tastatur (standardmäßig `Z` und `X`) Circles klicken, Slider verschieben und Spinner spinnen, um dadurch jedes zweite [Hit-Objekt](/wiki/Hit_object) zu treffen. Der Name kommt durch den Wechsel zwischen den Tasten. Dieser Stil ist am meisten verbreitet unter erfahrenen Spielern und verlangt eine der schwierigsten Anpassungen.
```

Comment:
The original is a bit weird here in my opinion. This is the best that I got. The focus is here on `Circles klicken, Slider verschieben und Spinner spinnen, um dadurch jedes zweite [Hit-Objekt](/wiki/Hit_object) zu treffen. Der Name kommt durch den Wechsel zwischen den Tasten.`